content/posts/being_becoming_liquidity_007.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Being, Becoming, and Liquidity"
+date: 2021-07-30T08:42:32
+draft: true
```

Comment:
@njcurtis3 you need to change this to `false` or remove the variable

Reply:
ok